### ISSUE TITLE
QForm autofocus doesn't fully select QInput

### ISSUE THREAD

**Issue by oskarmaxa** [quasarframework/quasar#5688]
**Steps to reproduce:**
1. Create a simple QForm with a QInput inside.
2. Set 'autofocus' on the QForm.
3. Load the page -> QInput appears selected, but text can't be entered until it's clicked, and caret isn't visible.

**Expected behavior:** QInput should be fully focused with caret visible and text enterable immediately.

It seems like the QInput's internal wrapper div gets focus rather than the input itself.

Note: The 'autofocus' prop works correctly when set directly on QInput.

**Quasar version:** 1.4.5

**Comment by pdanpdan** [comment]
Can you please make a codepen showing the problem?

**Comment by oskarmaxa** [comment]
I tried making both a jsfiddle and a codepen, but failed to demonstrate the issue there as the output iframe itself doesn't get focus on render.

On a regular webpage though, the following markup is sufficient to demonstrate the issue:
```
<q-form autofocus>
  <q-input filled />
</q-form>
```

For comparison, the following markup leads to the correct behavior:
```
<q-form>
  <q-input filled autofocus />
</q-form>
```

**Comment by pdanpdan** [comment]
Good, I can reproduce it, thank you.

**Comment by rstoenescu** [comment]
Hi,

Fix will be available in "quasar" v1.5.0
